src/rws_client.cpp

```diff
@@ -494,9 +534,9 @@ RWSClient::RWSResult RWSClient::evaluatePOCOResult(const POCOResult& poco_result
 
   if (log_.size() >= LOG_SIZE)
   {
-    log_.pop_front();
+    log_.pop_back();
   }
-  log_.push_back(poco_result);
+  log_.push_front(poco_result);
```

Comment:
I seem to have missed this in my initial review: this seems like quite a change. Was this incorrectly implemented before this change?

Reply:
It doesn't really change that much, since it will still keep the same information except it will be in the reverse order.

My reasoning was that it is more logically to keep the newest event at the front of the log (and it simplified for me when generating the log text).